public/style/vet.css

```
@@ -0,0 +1,798 @@
+}
+
+/* Texto da fala — VT323 lê bem em 1.25rem (20px). Antes em 1.05rem
+   (~16.8px) algumas palavras ficavam "esfumaçadas" pelo antialiasing
+   desencontrado com o grid bitmap. */
+.alice-bubble-text {
+  font-family: var(--font-family-pixel), monospace;
+  font-size: 1.25rem;
+  line-height: 1.5;
+  color: var(--vet-brown-deep);
+  margin: 0;
+  word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)